HW02/Dzmitry_Neverov/task3/line_search.rb

```diff
@@ -0,0 +1,13 @@
+class LineSearcher
+  attr_reader :some_lines
+
+  def initialize(some_lines)
+    @some_lines = some_lines
+  end
+
+  NEED_LOG = /.*Calling core with action.*/.freeze
```

Comment:
constants should be directly under the class name 
```suggestion
class LineSearcher
  NEED_LOG = /.*Calling core with action.*/.freeze
```